packages/react/src/react/components/compound/file-upload-field.tsx

```diff
@@ -318,13 +322,9 @@ const FileDisplayer = (props: {
   onRemoveSuccess?: (fileKey: string) => void
   onRemoveFail?: (reason: string) => void
 }) => {
-  // checking by fileKey is end with format type
-  const isImage =
-    props.fileKey.endsWith('.webp') ||
-    props.fileKey.endsWith('.png') ||
-    props.fileKey.endsWith('.jpg') ||
-    props.fileKey.endsWith('.jpeg') ||
-    props.fileKey.endsWith('.gif')
+  const baseUrl = process.env.NEXT_PUBLIC_AWS_IMAGE_URL
```

Comment:
This should be provided via props (which is drilled from user input)

Reply:
Please set this variable via `storageAdapter`. Also, please beware that this library is an open-source which will be used in several project. Please avoid side-effect code as much as possible.

Also, please beware that this open-source is global. So, please communicate in English for other contributor in the future.